src/Common/Database/QueryUtils.php

```diff
@@ -14,6 +14,24 @@
 
 class QueryUtils
 {
+    /**
+     * Function that will return an array listing
+     * of columns that exist in a table.
+     *
+     * @param   string  $table sql table
+     * @return  array
+     */
+    public static function listTableFields($table)
+    {
+        $sql = "SHOW COLUMNS FROM `" . \add_escape_custom($table) . "`";
```

Comment:
note that backtics won't protect against injection. If there is potential that this $table value may be modified (ie. not just hardcoded stuff), then would run it through the following whitelisting function:
https://github.com/openemr/openemr/blob/master/library/formdata.inc.php#L168

Reply:
Sounds good, I was just refactoring what the ORDataObject->persist method had originally.  I've changed it to be escape_table_name